docs/release/release_0_2_11.rst

```diff
@@ -0,0 +1,66 @@
+Announcement: napari 0.2.11
+===========================
+
+We're happy to announce the release of napari 0.2.11!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Highlights
+**********
+- Allow Points display properties to be set by point properties (#895)
+- Python 3.8 support (#878)
+
+Improvements
+************
+- Represent color as Nx4 array in Points  (#782)
+- Qt/Vispy connection & lambda refactor (#859)
+- Add PyQt5 tests to Linux and OSX CI (#867)
+- Bump tests to python 3.8, general tests fix (#878)
+- Improve slider step precision based on data range (#884)
+- Allow Points display properties to be set by point properties (#895)
+- Refactor add_* methods (#897)
+- Add _add_layer_data method and tests (#909)
+- Add Python 3.8 to PyPI tags (#917)
```

Comment:
not sure how relevant this is

Reply:
As noted above I like this list to be comprehensive, but I have created a new support category as you suggested so it does not become to hard to parse